packages/imperative/src/io/src/IO.ts

```diff
@@ -264,17 +264,38 @@ export class IO {
      * @returns {string} - input with removed newlines
      * @memberof IO
      */
-    public static processNewlines(original: string, lastByte?: number): string {
+    public static processNewlines<T extends string | Buffer>(original: T, lastByte?: number): T {
```

Comment:
If a type isn't explicitly provided here, it will only compile if `T extends string | Buffer`. The type of T would be inferred by the type of the `original` parameter, so I don't believe a default is necessary here.

Reply:
Thanks for clarifying.
I was under the same impression 😋